### PR TITLE
Fix: payment methods broken if method already exists

### LINKED ISSUE #3255: Fix being unable to pay for an item when you have a default payment method
How to reproduce:

1. Start the process of purchasing an item
2. When entering payment details, select that you want to set it as default
3. Continue as usual and place the order
4. Start the process of purchasing a second item
5. When on the payment details page, select the default payment and press continue (error)
6. Add another payment
7. Continue as usual and place the order
8. Start of the process of purchasing a third item
9. When on the payment details page, select ANY payment and press continue (error)

Screen Shot 2019-06-18 at 3 12 22 PM

### PR DESCRIPTION
**Pull request by LTiger14**
Closes #3255

### CODE REVIEW

File: projects/storefrontlib/src/cms-components/checkout/components/payment-method/payment-method.component.ts [hunk 1]
```diff
@@ -156,11 +156,14 @@ export class PaymentMethodComponent implements OnInit, OnDestroy {
...
   }): void {
-    this.getDeliveryAddressSub = this.checkoutDeliveryService
-      .getDeliveryAddress()
-      .subscribe(address => {
-        billingAddress = address;
-      });
+    if (!billingAddress) {
+      this.getDeliveryAddressSub = this.checkoutDeliveryService
+        .getDeliveryAddress()
+        .subscribe(address => {
```
Comment (WeizhengSap) [inline projects/storefrontlib/src/cms-components/checkout/components/payment-method/payment-method.component.ts]: If this method called multi times, then only the last subscription will be unsubscribed.

Comment (LTiger14) [reply projects/storefrontlib/src/cms-components/checkout/components/payment-method/payment-method.component.ts]: Fixed ;)

File: projects/storefrontlib/src/cms-components/checkout/components/payment-method/billing-address-form/billing-address-form.component.html [hunk 2]
```diff
@@ -90,6 +90,49 @@
         />
       </label>
     </div>
+    <div class="form-group col-md-6">
```
Comment (WeizhengSap) [inline projects/storefrontlib/src/cms-components/checkout/components/payment-method/billing-address-form/billing-address-form.component.html]: I checked the POST request of "payment/sop/response", and I found that the "billTo_State" is the same as "shipTo_State", even I select a different state.

File: projects/storefrontapp/src/environments/environment.ts [hunk 3]
```diff
@@ -5,5 +5,5 @@
 
 export const environment = {
   production: false,
-  occBaseUrl: '',
+  occBaseUrl: 'https://dev-com-17.accdemo.b2c.ydev.hybris.com:9002',
```
Comment (WeizhengSap) [inline projects/storefrontapp/src/environments/environment.ts]: please leave it empty

**Pull request merged**